alerts/sender_with_alert.sh

```diff
@@ -20,6 +37,27 @@ source "$1"
 
 ### FUNCTIONS ###
 
+# Function to get the tx cost from the explorer:
+# @param merkle_root
+# @param type_of_hash
+#   - Submission Transaction Hash
+#   - Response Transaction Hash
+function fetch_tx_cost() {
+  # Get the tx hash from the explorer
+  explorer_fetch_url="$EXPLORER_URL/batches/$1"
+  #echo "This is the fetch URL: $explorer_fetch_url"
+  tx_hash=$(curl -s $explorer_fetch_url | grep -C 5 "$2" | grep -oE '0x[[:alnum:]]{64}' | head -n 1)
```

Comment:
this sounds too fragile.
We should do a cast call to get the events related to the merkle root. 
more info on how to do this [here](https://book.getfoundry.sh/reference/cast/cast-logs?highlight=cast%20logs#cast-logs)

Reply:
Sure, I'll check that